src/closure.jl

```diff
@@ -196,6 +196,12 @@ function enclose(exorig::Expr, reserve_per, minbatchsize, per::Symbol, mod)
   innerloop = Symbol("##inner##loop##")
   rcombiner = Symbol("##split##recombined##")
 
+  if threadlocal === :()
+    threadlocal_name = :()
+  else
+    threadlocal_name = threadlocal.args[1]
+  end
```

Comment:
```suggestion
  threadlocal_name::Symbol = if threadlocal === Symbol("#reserved#polyester#thread#local#storage#")
    Symbol("#reserved#polyester#thread#local#storage#")
  else
    threadlocal.args[1]
  end
```

Reply:
You could of course define something like 
```julia
const NOTHREADLOCAL = Symbol("#reserved#polyester#thread#local#storage#")
```
and then use this variable everywhere. It'd probably look better than a deliberately unwieldy symbol.